pgml-extension/pgml_rust/src/orm/estimator.rs

```diff
@@ -431,3 +432,82 @@ impl Estimator for BoosterBox {
         self.contents.predict(&features).unwrap()[0]
     }
 }
+
+/// A wrapper around a Scikit estimator.
+/// The estimator is a Python object and can only be used
+/// inside Python::with_gil.
+pub struct SklearnBox {
+    pub contents: Box<Py<PyAny>>,
+}
+
+impl Drop for SklearnBox {
```

Comment:
Why do you need to implement this manually? Looks like fun...

Reply:
Just GIL things.